en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+
+:::note
+For instructions on using the manual PID tuning see:
+- [Multicopter PID Tuning Guide](../config_mc/pid_tuning_guide_multicopter_basic.md) (Manual/Simple)
+- [Multicopter PID Tuning Guide](../config_mc/pid_tuning_guide_multicopter.md) (Advanced/Detailed)
+- [Fixed-Wing PID Tuning Guide](../config_fw/pid_tuning_guide_fixedwing.md)
+:::
+
+<!-- 
+Overview of the auto-tuning workflow
+
+@[youtube](https://youtu.be/uHVlIIydcpg)
+-->
+
+## Pre-tuning Test
+
+The vehicle must be able to fly and adequately stabilize itself before running auto-tune.
+
+:::note
+During [Airframe Setup](../config/airframe.md) you should have selected the frame that most closely matches your vehicle.
+This will usually be tuned well enough to fly, and it _may_ also be sufficiently well tuned to run autotuning.
+:::
+
+To make sure the vehicle is stable enough for auto-tuning:
+
+1. Perform a normal preflight safety checklist to ensure the flight zone is clear and has enough space.
+2. Takeoff and prepare for the test
+   - **Multicopters:** Take off and hover at 1m above ground in Position mode, Altitude mode or Hold mode.
+   - **Fixed-wing:** Take off and fly at cruise speed in Position mode or Alitude mode.
+2. Use the RC transmitter roll stick to perform the following maneuver, tilting the vehicle just a few degrees: _roll left > roll right > center_ (The whole maneuver should take about 3 seconds).
+   The vehicle should stabilise itself within 2 oscillations.
+3. Repeat the maneuver, tilting with larger amplitudes at each attempt.
+   If the vehicle can stabilise itself within 2 oscillations at ~20 degrees move to the next step.
+4. Repeat the same maneuvers but on the pitch axis.
+   A above, start with small angles and confirm that the vehicle can itself within 2 oscillations before increasing the tilt.
+
+If the drone can stabilize itself within 2 oscillations it is ready for the auto-tuning procedure.
+
+If not, go to the [troubleshooting](#troubleshooting) section, which explains the minimal manual tuning to prepare the vehicle for auto-tuning.
+
+
+### Auto-tuning procedure
+
+The auto-tuning sequence must be performed in a **safe flight zone, with enough space**.
+For best results, it is recommended to fly in calm weather conditions.
+
+:::note
+The sequence can be aborted at any time by the operator by moving the roll/pitch stick on the RC controller.
+:::
+
+1. Perform the [pre-tuning test](#pre-tuning-test) detailed previously
+3. Takeoff using RC control and prepare for test:
+   - **Multicopters:** Takeoff using the remote controller in altitude mode.
+     Hover the vehicle at a safe distance and at a few meters above ground (between 4 and 20m).
+     :::note
+     If the drone is known to be stable in position controlled mode, you can 
+     - run auto-tuning in position mode and also takeoff using takeoff mode.
+     - use takeoff mode to take off.
+     :::
+      
+   - **Fixed-wing:** Once flying at cruise speed, activate auto mode.
```

Comment:
Do you mean hold mode? Auto is "mission".

Reply:
Yes, hold mode

Reply:
```suggestion
   - **Fixed-wing:** Once flying at cruise speed, activate [Hold mode](../flight_modes/hold.md).
```